File: Objective-C-Project/GStack-ObjectiveC-SDK-StoryBoard/TestStoryBoard/BracketTableViewCell.m
//
//  BracketTableViewCell.m
//  TestStoryBoard
//
//  Created by 谢乾坤 on 2/19/16.
//  Copyright © 2016 QiankunXie. All rights reserved.
//

#import "BracketTableViewCell.h"

@implementation BracketTableViewCell

- (void)awakeFromNib {
    // Initialization code
}

- (void)setSelected:(BOOL)selected animated:(BOOL)animated {
    [super setSelected:selected animated:animated];

    // Configure the view for the selected state
}

- (void)setFrame:(CGRect)frame {
    frame.origin.x += 15;
    frame.size.width -= 2 * 15;
    [super setFrame:frame];
}

@end

File: Swift-Project/GStack-Swift-SDK/GStack-Swift-SDK/GStack-Swift-SDK/BracketTableViewCell.swift
//
//  BracketTableViewCell.swift
//  GStack-Swift-SDK
//
//  Created by 谢乾坤 on 3/7/16.
//  Copyright © 2016 QiankunXie. All rights reserved.
//

import UIKit

class BracketTableViewCell: UITableViewCell {
    
    
    @IBOutlet weak var tournamentName: UILabel!
    
    @IBOutlet weak var tournamentStyle: UILabel!
    
    @IBOutlet weak var tournamentCategory: UILabel!
    
    @IBOutlet weak var tournamentQuestionNum: UILabel!
    

    override func awakeFromNib() {
        super.awakeFromNib()
        // Initialization code
    }

    override func setSelected(selected: Bool, animated: Bool) {
        super.setSelected(selected, animated: animated)

        // Configure the view for the selected state
    }
    /*
    override func layoutSubviews() {
        self.bounds = CGRectMake(self.bounds.origin.x, self.bounds.origin.y, self.bounds.size.width - 30, self.bounds.size.height)
        super.layoutSubviews()
    }
*/
}
